src/HSE/Util.hs

```diff
@@ -15,7 +15,8 @@ import System.FilePath
 import HSE.Type
 import Data.Functor
 import Prelude
-
+import qualified GHC.LanguageExtensions.Type()
```

Comment:
Why import this qualified and with nothing in scope. Drop the qualified, and either drop the import entirely, or comment which orphan instance requires it.

Reply:
Dropping the import. Turns out not required.